### PR TITLE
[RPS-328] Dataset without publication will show 404

### PR DESCRIPTION
**Pull request by jacsamell**
Also other housekeeping changes.

- Updated dependancy versions
- Turned off diagnostic logging
- Turned off auto-export by default
- Moved random doc in parent folder to acceptance test folder
- Deleted some taxonomy tests that are no longer valid

### CODE REVIEW

File: acceptance-tests/src/test/resources/features/site/datasets.feature [hunk 1]
```diff
@@ -73,6 +73,10 @@ Feature: As a consumer I need to be able to navigate to publication data sets
         Given I navigate to the "upcoming publication dataset" dataset page
         Then I should see the page not found error page
 
+    Scenario: Hide dataset if it doesn't have a parent publication
+        Given I navigate to the "dataset without publication" dataset page
+        Then I should see the page not found error page
```
Comment (ziemsky) [inline acceptance-tests/src/test/resources/features/site/datasets.feature]: Would read slightly better if 'page not found' were embraced in quotes. (also applies to the scenario above).

File: repository-data/development/src/main/resources/hcm-config/main.yaml [hunk 2]
```diff
@@ -2,7 +2,7 @@
 definitions:
   config:
     /hippo:configuration/hippo:modules/autoexport/hippo:moduleconfig:
-      autoexport:enabled: true
+      autoexport:enabled: false
```
Comment (ziemsky) [inline repository-data/development/src/main/resources/hcm-config/main.yaml]: Yay! :smile:

File: repository-data/development/src/main/resources/hcm-config/hst/hosts.yaml [hunk 3]
```diff
@@ -48,4 +48,4 @@ definitions:
         hst:cmslocation: http://hippo-authoring.int:8080/cms
         hst:defaultport: 8080
         jcr:primaryType: hst:virtualhostgroup
-      hst:diagnosticsenabled: true
+      hst:diagnosticsenabled: false
```
Comment (ziemsky) [inline repository-data/development/src/main/resources/hcm-config/hst/hosts.yaml]: :+1:

File: repository-data/development/src/main/resources/hcm-content/content/documents/corporate-website/publication-system/acceptance-tests/another-upcoming-publication.yaml [hunk 4]
```diff
@@ -0,0 +1,111 @@
+---
+/content/documents/corporate-website/publication-system/acceptance-tests/another-upcoming-publication:
```
Comment (wojtek-oledzki) [inline repository-data/development/src/main/resources/hcm-content/content/documents/corporate-website/publication-system/acceptance-tests/another-upcoming-publication.yaml]: :+1: for cleaning that up

File: Makefile [hunk 5]
```diff
@@ -67,7 +67,7 @@ format-yaml:
 
 ## Update maven dependency versions
 update-dependencies:
-	mvn verify versions:use-latest-versions versions:update-properties versions:commit -U
+	mvn verify versions:update-parent versions:use-latest-versions versions:update-properties versions:commit -U -DskipTests=true
```
Comment (wojtek-oledzki) [inline Makefile]: bravo!

File: acceptance-tests/src/test/resources/features/site/datasets.feature (hunk 1 above)
Comment (jacsamell) [reply acceptance-tests/src/test/resources/features/site/datasets.feature]: Fixed 😄

**Pull request merged**